packages/synapse-react-client/src/components/dataaccess/RejectDataAccessRequestModal.integration.test.tsx

```diff
@@ -24,7 +26,10 @@ const props: RejectDataAccessRequestModalProps = {
   onClose: jest.fn(),
 }
 
-const onServerReceivedUpdate = jest.fn()
```

Comment:
Generally trying to remove these mocks that are injected into the MSW handler. While it's nice to verify that the server actually received the expected request body, it makes these handlers harder to extract and reuse.